CryptoAnalysis/src/main/java/crypto/typestate/FiniteStateMachineToTypestateChangeFunction.java

```diff
@@ -36,11 +41,7 @@ public FiniteStateMachineToTypestateChangeFunction(SootBasedStateMachineGraph fs
 				} else {
 					// This code was added to detect unidentified outlying cases affected by the changes made for issue #47.
 					if (analyzedType != m.getDeclaringClass().getType()){
-                        try {
```

Comment:
WTF isn't there an easier way in java to get the stacktrace? :D

Reply:
@AnakinSklavenwalker  I was unsure of what to do here, as I was not understanding why the stack trace was being printed. Therefore, I am just logging an error message. Shall I print the stack trace as before, or leave it as it is now.